ci-operator/step-registry/openshift-qe/installer/azure/ipi/deprovision/openshift-qe-installer-azure-ipi-deprovision-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
+  as: openshift-qe-installer-azure-ipi-deprovision
+  steps:
+    - chain: ipi-deprovision
```

Comment:
Since this has only one step I think we don't have to make a separate chain for this. Can use the ipi-deprovision directly in the main workflow as a post step

Reply:
@paigerube14 You're right.I'll remove this chain and directly use ipi-deprovision chain.